packages/navis_ui/lib/src/icons/genesis_assets.dart

```diff
@@ -19,62 +19,92 @@ class GenesisAssets {
   GenesisAssets._();
 
   static const _kFontFam = 'GenesisAssets';
-  static const _kFontPkg = 'navis_ui';
+  static const String? _kFontPkg = 'navis_ui';
 
   static const IconData sentient =
       IconData(0xe800, fontFamily: _kFontFam, fontPackage: _kFontPkg);
+
   static const IconData simaris =
       IconData(0xe801, fontFamily: _kFontFam, fontPackage: _kFontPkg);
+
   static const IconData solaris =
       IconData(0xe802, fontFamily: _kFontFam, fontPackage: _kFontPkg);
+
   static const IconData standing =
       IconData(0xe803, fontFamily: _kFontFam, fontPackage: _kFontPkg);
+
   static const IconData weekly =
       IconData(0xe804, fontFamily: _kFontFam, fontPackage: _kFontPkg);
-  static const IconData wfcdLogo =
-      IconData(0xe81c, fontFamily: _kFontFam, fontPackage: _kFontPkg);
+
+  static const IconData narmer =
```

Comment:
did you already add narmer on our assets on the website too?

Reply:
Not yet, I don't like how the auto stuff did it. Gonna manually trace it tonight.

It's fine on mobile because it's a small enough defect nobody will notice but very noticeable on a bigger screen